schema/descriptor.schema.json

```diff
@@ -257,25 +296,50 @@
                         "id": "http://github.com/boutiques/boutiques-schema/input/max-list-entries",
                         "description": "Specify the maximum number of entries in the list. May only be used with List type inputs.",
                         "type": "number"
+                    },
+                    "absolute-path": {
+                        "id": "http://github.com/boutiques/boutiques-schema/input/absolute-path",
+                        "description": "Specifies that this input must be given as an absolute path. Only specifiable for File type inputs. Cannot be true if relative-path is true.",
+                        "type": "boolean"
+                    },
+                    "relative-path": {
+                        "id": "http://github.com/boutiques/boutiques-schema/input/relative-path",
```

Comment:
I prefer a single `use-absolute-path` as well. 

One minor thing is the semantics of this property: does it mean that an input must be absolute (i.e. it just checks it) vs the platform must transform it, or is the interpretation up to the platform?